src/store/modules/user/index.ts

```diff
@@ -10,8 +10,11 @@ const userModule: Module<UserState, RootState> = {
     state: {
       token: '',
       current: null,
-      selectedBrand: '',
       instanceUrl: '',
+      currentEComStore: {
+        productStoreId: "",
```

Comment:
Keep it empty, we will not support None here

Reply:
ok sir